src/libraries/System.Text.Json/src/PACKAGE.md

```diff
@@ -1,11 +1,258 @@
+//datesAvailable JSON =["2019-08-01T00:00:00", "2019-08-02T00:00:00"]
+
+// Get an array element value from a JsonArray.
+JsonNode firstDateAvailable = datesAvailable[0]!;
+Console.WriteLine($"Type={firstDateAvailable.GetType()}");
+Console.WriteLine($"JSON={firstDateAvailable.ToJsonString()}");
+//output:
+//Type = System.Text.Json.Nodes.JsonValue`1[System.Text.Json.JsonElement]
+//JSON = "2019-08-01T00:00:00"
+
+// Get a typed value by chaining references.
+int coldHighTemperature = (int)forecastNode["TemperatureRanges"]!["Cold"]!["High"]!;
+Console.WriteLine($"TemperatureRanges.Cold.High={coldHighTemperature}");
+//output:
+//TemperatureRanges.Cold.High = 20
+
+// Parse a JSON array
+var datesNode = JsonNode.Parse(@"[""2019-08-01T00:00:00"",""2019-08-02T00:00:00""]");
+JsonNode firstDate = datesNode![0]!.GetValue<DateTime>();
+Console.WriteLine($"firstDate={ firstDate}");
+//output:
+//firstDate = "2019-08-01T00:00:00"
+```
+
+Using the reader/writer
+```csharp
+/*
+  SharpLab tools in Run mode:
+    • value.Inspect()
+    • Inspect.Heap(object)
+    • Inspect.Stack(value)
+    • Inspect.MemoryGraph(value1, value2, …)
+*/
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+var writerOptions = new JsonWriterOptions
+{
+    Indented = true
+};
+
+using var stream = new MemoryStream();
+using var writer = new Utf8JsonWriter(stream, writerOptions);
+
+writer.WriteStartObject();
+writer.WriteString("date", DateTimeOffset.Parse("8/2/2023 9:00 AM"));
+writer.WriteNumber("temp", 42);
+writer.WriteEndObject();
+writer.Flush();
+
+var jsonBytes = stream.ToArray();
+string json = Encoding.UTF8.GetString(jsonBytes);
+Console.WriteLine(json);
+// {
+//   "date": "2023-08-02T09:00:00+00:00"
+//   "temp": 42
+// }
+
+var readerOptions = new JsonReaderOptions
+{
+    AllowTrailingCommas = true,
+    CommentHandling = JsonCommentHandling.Skip
+};
+var reader = new Utf8JsonReader(jsonBytes, readerOptions);
+
+while (reader.Read())
+{
+    Console.Write(reader.TokenType);
+
+    switch (reader.TokenType)
+    {
+        case JsonTokenType.PropertyName:
+        case JsonTokenType.String:
+            {
+                string? text = reader.GetString();
+                Console.Write(" ");
+                Console.Write(text);
+                break;
+            }
+
+        case JsonTokenType.Number:
+            {
+                int intValue = reader.GetInt32();
+                Console.Write(" ");
+                Console.Write(intValue);
+                break;
+            }
+
+            // Other token types elided for brevity
+    }
+    Console.WriteLine();
+}
+// StartObject
+// PropertyName date
+// String 2023-08-02T09:00:00+00:00
+// PropertyName temp
+// Number 42
+// EndObject
+```
+
+## Key Features
+
+<!-- The key features of this package -->
+
+TODO
+
+## Main Types
+
+<!-- The main types provided in this library -->
+
+The main types provided by this library are:
+
+* `System.Text.Json.JsonSerializer`
```

Comment:
`JsonSerializerOptions` is fairly pervasive and important. Should we include it here or does it get covered by the "JsonSerializer" umbrella?

Reply:
Yep - please add whichever types you think folks will think of as entry point types.  No need to ask - just add them.